projects/canopy/src/lib/table/table-cell/table-cell.component.scss

```diff
@@ -21,16 +21,26 @@
 
 .lg-table-cell__content {
   margin-left: auto;
+  white-space: nowrap;
+
+  .lg-table-expanded-detail {
```

Comment:
Should this selector live in here? I'm just wondering to be more future safe if it should live the scss file for the expanded detail component? (which might need creating).

Where possible I think we should try and stop the components being aware of one another.

Reply:
Yes, you're right 🤦  updated now